modules/token-overview.adoc

```diff
@@ -0,0 +1,66 @@
+:_content-type: CONCEPT
+[id="token-overview"]
+= Introduction to {productname} OAuth 2.0 tokens
+
+The {productname} OAuth2 token system provides a secure, standards-based method for accessing {productname}'s API and other relevant resources. The {productname} OAuth2 token-based approaches provides a secure method for handling authentication and authorization for complex environments. Compared to more traditional API tokens, {productname}'s OAuth2 token system offers the following enhancements:
+
+* Standards-based security, which adheres to the link:https://oauth.net/2/[OAuth 2.0 protocol].
+* Revocable access by way of deleting the application in which the OAuth2 token exists. 
+* Fine-grained access control, which allows {productname} administrators the ability to assign specific permissions to tokens.
+* Delegated access, which allows third-party applications and services to act on behalf of a user.
+* Future-proofing, which helps ensure that {productname} remains compatible with other services, platforms, and integrations.
+
+The following sections provide more tails about the token types available with {productname}.
+
+[id="token-types"]
+== {productname} token types
+
+{productname} primarily supports two types of tokens: OAuth2 access tokens and robot account tokens. A third token type, an _OCI referrers access token_, that is required to list OCI referrers of a manifest under a repository, is also available when warranted.
+
+[discrete]
+[id="oauth2-access-tokens"]
+=== OAuth2 access tokens 
+
+OAuth 2 access tokens enable user-authenticated access to the {productname} API, suitable for applications that require user identity verification. These tokens are obtained through an OAuth2 authorization process, where a {productname} administrator generates a token on behalf of themselves or another user to access {productname} resources. These tokens provide scope-based access and specify what resources and actions the token holder is allowed. Currently, the following options can be selected for a token holder:
```

Comment:
What does it mean `resources`? I'm asking because if `resources` are repositories and teams and robot accounts and other stuff inside Quay, then Oauth2 tokens do not provide access to those. Oauth2 tokens only provide access to specific API endpoints, via defined scopes. Access to resources themselves is governed by RBAC, you can make an action on a resource provided that you have control over that resource. For instance, you can create a repository in a namespace if you have creator privileges on that namespace. This holds true even if your API token contains `repo:admin` scope.